arenadata/scripts/test_pitr_gpdb.sh

```diff
@@ -122,8 +122,15 @@ do
    check_backup $i
 done
 
+# Determine version
+GP_VERSION_NUM=$(pg_config --gp_version | cut -c 11)
+
 # Creating a distributed restore point..."
-psql -c "create extension gp_pitr;"
+if [ $GP_VERSION_NUM -le 6 ]; then
```

Comment:
Could you please add comment describing why this condition is needed. How it was done in PR description.